.github/workflows/ssh-script-example.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Google LLC
```

Comment:
Same question - do we want this to actually run as part of our CI script or is this more "documentation"?